src/Raven.Server/Documents/Indexes/Persistence/IndexReadOperationBase.cs

```diff
@@ -24,8 +24,7 @@ protected IndexReadOperationBase(Index index, Logger logger, QueryBuilderFactori
         {
             QueryBuilderFactories = queryBuilderFactories;
 
-
-            if (_logger.IsInfoEnabled && query != null)
+            if (query != null)
```

Comment:
What if logging is completely disabled? So neither info nor operations level is enabled